scripts/init-mptcp.sh

```diff
@@ -0,0 +1,33 @@
+echo "Setting up MPTCP..."
+
+vagrant ssh client <<EOF
+    # Sometimes the ip addresses aren't assigned by virtualbox, so do it manually
+    sudo ip addr add 192.168.56.100/24 dev eth1
+    sudo ip addr add 192.168.57.100/24 dev eth2
+    sudo ip addr add 192.168.58.100/24 dev eth3
+
+    # Configure MPTCP
+    sudo sysctl net.mptcp.enabled=1
+    sudo ip mptcp limits set subflow 2
+    sudo ip mptcp limits set add_addr_accepted 2
+    sudo ip mptcp endpoint add 192.168.57.100 dev eth2 subflow fullmesh
+    sudo ip mptcp endpoint add 192.168.58.100 dev eth3 subflow fullmesh
+EOF
+
+vagrant ssh server <<EOF
+    # Sometimes the ip addresses aren't assigned by virtualbox, so do it manually
+    sudo ip addr add 192.168.56.101/24 dev eth1
+    sudo ip addr add 192.168.57.101/24 dev eth2
+    sudo ip addr add 192.168.58.101/24 dev eth3
+
+    # Configure MPTCP
+    sudo sysctl net.mptcp.enabled=1
+    sudo ip mptcp limits set subflow 2
+    sudo ip mptcp endpoint add 192.168.57.101 dev eth2 signal
+    sudo ip mptcp endpoint add 192.168.58.101 dev eth3 signal
+
+    # Enable MPTCP on nginx
+    # sudo mptcpize enable nginx
```

Comment:
why commenting it?
(also, I don't remember if you need to reload the systemd daemon and/or restart the nginx service)

Reply:
I tried to do experiments with NGINX, but removed it because there were permission issues. This is an artifact of it, I'll remove it